src/components/statusFilter.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import { twJoin } from "tailwind-merge";
+
+interface OptionStyle {
+  option: string;
+  status: string;
+  activeClass: string;
+  inactiveClass: string;
+}
+
+interface StatusFilterProps {
+  onStatusChange: (status: string) => void;
```

Comment:
I suggest you change the string type to the status type mentioned above.
```suggestion
  onStatusChange: (status: Status) => void;
```

Reply:
we did it together